### PR TITLE
fzf-make: init at 0.6.0

### PR DESCRIPTION
**Pull request by figsoda**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by AndersonTorres** [comment]
Duplicated:

https://github.com/NixOS/nixpkgs/pull/250019

File: pkgs/development/tools/misc/fzf-make/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      --suffix PATH : ${lib.makeBinPath [ gnumake ]}
+  '';
+
+  meta = with lib; {
+    description = "Fuzzy finder for Makefile";
+    homepage = "https://github.com/kyu08/fzf-make";
+    changelog = "https://github.com/kyu08/fzf-make/releases/tag/${src.rev}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ figsoda ];
+    mainProgram = "fzf-make";
```
Comment (winterqt) [inline pkgs/development/tools/misc/fzf-make/default.nix:36]: Does this not default to pname?

**Comment by figsoda** [comment]
> Duplicated:
> 
> #250019

Well, this one was opened first

File: pkgs/development/tools/misc/fzf-make/default.nix (hunk 1 above)
Comment (figsoda) [reply pkgs/development/tools/misc/fzf-make/default.nix:36]: It won't be in the future for `lib.getExe`: #246386

**Comment by Sigmanificient** [comment]
> > Duplicated:
> > #250019
> 
> Well, this one was opened first



> 

Could you add me as Co-Authored-By too?

**Comment by figsoda** [comment]
> > > Duplicated:
> > > #250019
> > 
> > 
> > Well, this one was opened first
> 
> > 
> 
> Could you add me as Co-Authored-By too?

done

**Comment by Sigmanificient** [comment]
> > > > Duplicated:
> > > > #250019
> > > 
> > > 
> > > Well, this one was opened first
> > 
> > 
> > > 
> > 
> > 
> > Could you add me as Co-Authored-By too?
> 
> done

Closed mine!

**Comment by AndersonTorres** [comment]
> > Duplicated:
> > 
> > #250019
> 
> Well, this one was opened first

The newest ones appeared first to me. Sorry for the hassle.

**Comment by figsoda** [comment]
No worries!

File: pkgs/development/tools/misc/fzf-make/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/development/tools/misc/fzf-make/default.nix:36]: We need something like a "nixpkgs news dashboard" for these things.

Comment (AndersonTorres) [reply pkgs/development/tools/misc/fzf-make/default.nix:36]: We need something like a "nixpkgs news dashboard" for these things.

**Comment by Sigmanificient** [comment]
@figsoda There is a missing dependency for `bat`, what should we do about it?

**Comment by AndersonTorres** [comment]
> @figsoda There is a missing dependency for `bat`, what should we do about it?

Open a PR.

**Comment by figsoda** [comment]
#250187

**Pull request merged**